tests/unit/mock-api-data/terms-cat.json

```diff
@@ -0,0 +1,342 @@
+[
+  {
+    "term": "cat",
```

Comment:
@rhyolight I did not think at all of including a mock data file... this answers a lot of my questions about mocking out API calls.